src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Abstractions/Message Center/MessagePage.cs

```diff
@@ -0,0 +1,153 @@
+﻿/*
+ Copyright Airship and Contributors
+*/
+
+using System;
+
+using Xamarin.Forms;
+
+namespace UrbanAirship.NETStandard.MessageCenter
+{
+    /// <summary>
+    /// ContentPage used for displaying Message Center Messages.
+    /// </summary>
+    public partial class MessagePage : ContentPage
+    {
+        /// <summary>
+        /// Gets and sets the message ID.
+        /// </summary>
+        /// <value>The message ID.</value>
+        public string MessageId { get; set; }
+
+        /// <summary>
+        /// Event handler invoked when the message has loaded.
+        /// </summary>
+        public event EventHandler<MessageLoadedEventArgs> Loaded;
+
+        /// <summary>
+        /// Event handler invoked whe  the message load fails.
+        /// </summary>
+        public event EventHandler<MessageLoadFailedEventArgs> LoadFailed;
+
+        /// <summary>
+        /// Event handler invoked when the message is closed.
+        /// </summary>
+        public event EventHandler<MessageClosedEventArgs> Closed;
+
+        /// <summary>
+        /// MessagePage constructor.
+        /// </summary>
+        public MessagePage()
+        {
+        }
+
+        //@cond IGNORE
+        public void OnRendererLoaded(string messageId)
```

Comment:
These methods were an attempt to bridge the more c-sharp-y EventHandlers in the MessagePage with the callbacks in the renderers. C# is very strict about the invocation of event handlers and doesn't allow them to be invoked outside of the scope they're defined in. Since renders can access the page at runtime but not the other way around, this seems like a reasonable compromise.